packages/docs/src/components/Layout.tsx

```diff
@@ -68,12 +68,15 @@ const Layout = ({ children, frontmatter, location, theme, tableOfContentsData }:
           <article className="ds-u-md-display--flex ds-u-padding-x--3 ds-u-sm-padding-x--6 ds-u-sm-padding-bottom--6 ds-u-sm-padding-top--1 ds-u-padding-bottom--3 page-content">
             <div className="page-content__content ds-l-lg-col--9 ds-u-padding-left--0">
               <div className="ds-u-display--block ds-u-lg-display--none">
-                <TableOfContentsMobile items={tableOfContentsData || []} />
+                <TableOfContentsMobile
+                  title={frontmatter.title}
+                  items={tableOfContentsData || []}
+                />
               </div>
               {children}
             </div>
             <div className="ds-l-lg-col--3 ds-u-display--none ds-u-lg-display--block">
-              <TableOfContents items={tableOfContentsData || []} />
+              <TableOfContents title={frontmatter.title} items={tableOfContentsData || []} />
```

Comment:
I forgot about this... Ugh... I added a `title` prop to the ToC components to make it easier to create the GH discussions link. 

I ultimately removed the GH discussions link from the ToC components because that work was part of a different ticket and forgot to remove this prop reference in the process.

Reply:
No worries! You can go ahead and remove all these changes in `TableOfContents` & `TableOfContentsMobile` as I've implemented these (along with the feedback page link, `/feedback` in my PR for the discussions link: #2024